docs/docs/getting-started.md

```diff
@@ -81,152 +95,53 @@ ignorePaths:
 
 These configuration files do three things:
 
-1. Define the custom dictionary `project-words`.
+1. Define the custom dictionary of `project-words`.
 1. Tell the spell checker to use the custom dictionary.
 1. Tell the spell checker to ignore any files inside of `node_modules` and the file `project-words.txt`.
 
-## 2. Add words to the project dictionary
+Finally, create the dictionary file as well.
 
-It might take a few iterations to get fully setup, but the process in the same.
+```sh
+touch project-words.txt
+```
 
-Steps:
+The dictionary is simply a text file that contains each word on a separate line.
 
-1. Create the dictionary file
+## Add words to the project dictionary
 
-   ```sh
-   touch project-words.txt
-   ```
+Now, we have to start filtering out the false positives. It might take a few iterations to get everything filtered out.
 
-1. Choose a set of files to start with, like all Markdown files, `**/*.md` and run the spell checker.
+1. Choose a set of files to start with, like all Markdown files. Then, run CSpell.
 
    ```sh
    cspell "**/*.md"
    ```
 
-1. Look for any directories that need to be ignored and add them to `ignorePaths`. Example:
+   This will give you a baseline idea about what to do in the next steps.
+
+1. Based on the results on the spell check, you might see that there are some false positives from files that you don't care about (like transpiled files in a "dist" directory). If this is the case, then add those paths to the `ignorePaths` section of the configuration file. For example:
 
-   - `"bin"` - to ignore any directory / file called `bin`.
-   - `"translations/**"` - to ignore all files under the `translations` directory.
-   - `"packages/*/dist"` - to ignore the `dist` directory in each _package_.
+   - `"bin"` - Ignores any directory or file called `bin`.
+   - `"translations/**"` - Ignores all files under the `translations` directory.
+   - `"packages/*/dist"` - Ignores the `dist` subdirectory in each `package` directory.
 
-   Once you have finished identifying directories and files to be ignored, it is now time to add words to the custom dictionary.
+1. From here, run CSpell again, and depending on how many false positives there are, you can proceed in two different ways. First, if there are only a handful of false positives remaining, then you can just manually add them to the dictionary file, and then you are done!
 
-1. Have CSpell populate it with the words from your project.
+1. If there are a lot of false positives, then you can programmatically add every misspelled word to the custom dictionary:
 
    ```sh
-   echo "# New Words" >> project-words.txt
    cspell --words-only --unique "**/*.md" | sort --ignore-case >> project-words.txt
    ```
 
-   This will append all new issues to the end of `project-words.txt`
-
-1. Review the words in `project-words.txt` to check for any actual misspellings and remove them (the spell checker already thinks they are wrong).
-
-1. Fix spelling issues.
```

Comment:
I don't quite follow, sorry. The 2 lines that you highlighted here were removed because this information is already included in the following passage:

> 1. Go through all of the words that were added to `project-words.txt` to ensure that there were no real spelling errors that were copied over. If you found a real misspelled word, then delete it from the dictionary, and fix the spelling error in the source code.
> 1. Repeat the process with the other file types you want to check.

This is a more verbose / more clear way to explain what I believe that the original author was trying to convey.